pdns/recursordist/settings/cxxsupport.cc

```diff
@@ -119,18 +122,172 @@ void pdns::settings::rec::oldStyleAllowFileToBridgeStruct(const std::string& fil
+    if (plusis != string::npos) {
+      auto val = line.substr(plusis + 2);
+      boost::trim(val);
+      result.emplace_back(val);
+    }
+  }
+
+  rust::string key = "allow_from";
+  rust::string filekey = "allow_from_file";
+  if (filename == "allow-notify-from") {
+    key = "allow_notify_from";
+    filekey = "allow_notify_from_file";
+  }
+  const auto yaml = pdns::rust::settings::rec::allow_from_to_yaml_string_incoming(key, filekey, result);
+
+  string yamlfilename = apiDir;
+  yamlfilename.append("/").append(filename).append(".yml");
+  string tmpfilename = yamlfilename + ".tmp";
+  ofstream ofconf(tmpfilename);
+  if (!ofconf) {
+    int err = errno;
+    log->error(Logr::Error, err, "Cannot open for file for writing YAML", "to", Logging::Loggable(tmpfilename));
+    throw runtime_error("YAML Conversion");
+  }
+  ofconf << "# Generated by pdns-recursor REST API, DO NOT EDIT" << endl;
+  ofconf << yaml << endl;
+  ofconf.close();
+  if (rename(path.c_str(), (path + ".converted").c_str()) != 0) {
+    int err = errno;
+    log->error(Logr::Error, err, "Rename failed", "file", Logging::Loggable(path), "to", Logging::Loggable(path + ".converted"));
+    unlink(tmpfilename.c_str());
+    throw runtime_error("YAML Conversion");
+  }
+
+  if (rename(tmpfilename.c_str(), yamlfilename.c_str()) != 0) {
+    int err = errno;
+    log->error(Logr::Error, err, "Rename failed", "file", Logging::Loggable(tmpfilename), "to", Logging::Loggable(yamlfilename));
+    rename((path + ".converted").c_str(), path.c_str());
+    throw runtime_error("YAML Conversion");
+  }
+  log->info(Logr::Notice, "Converted to YAML", "file", Logging::Loggable(path), "to", Logging::Loggable(yamlfilename));
+}
+
+static void fileCopy(const string& src, const string& dst, Logr::log_t log)
+{
+  ifstream ifconf(src);
+  if (!ifconf) {
+    log->info(Logr::Error, "Cannot open for file for reading", "file", Logging::Loggable(src));
+    throw runtime_error("YAML Conversion");
+  }
+  ofstream ofconf(dst);
+  if (!ofconf) {
+    log->info(Logr::Error, "Cannot open for file for writing YAML", "to", Logging::Loggable(dst));
+    throw runtime_error("YAML Conversion");
+  }
+  for (;;) {
+    auto character = ifconf.get();
+    if (ifconf.eof()) {
+      break;
+    }
+    if (ifconf.bad()) {
+      int err = errno;
+      log->error(Logr::Error, err, "Error reading", "to", Logging::Loggable(src));
+      throw runtime_error("YAML Conversion");
+    }
+    ofconf.put(static_cast<char>(character));
+    if (ofconf.bad()) {
+      int err = errno;
+      log->error(Logr::Error, err, "Error writing YAML", "to", Logging::Loggable(dst));
+      throw runtime_error("YAML Conversion");
+    }
+  }
+  ofconf.close();
+  ifconf.close();
+}
+static void possiblyConvertForwardsandAuths(const std::string& includeDir, const std::string& apiDir, Logr::log_t log)
+{
+  std::vector<std::string> forwAndAuthFiles{};
+  ::arg().gatherIncludes(includeDir, "..conf", forwAndAuthFiles);
```

Comment:
Unless I'm mistaken, the suffix is wrong:
```suggestion
  ::arg().gatherIncludes(includeDir, ".conf", forwAndAuthFiles);
```

Reply:
Nope, the recursor generated file names in the api dir for API managed forward and auth file are using double dots.

Reply:
Well, today I learned.. :)